src/Workspaces/VisualBasic/Portable/Simplification/VisualBasicSimplificationService.Expander.vb

```diff
@@ -622,14 +622,15 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Simplification
                     If DirectCast(symbol, IMethodSymbol).TypeArguments.Length <> 0 Then
                         Dim typeArguments = DirectCast(symbol, IMethodSymbol).TypeArguments
 
-                        newNode = SyntaxFactory.GenericName(
+                        Dim genericName = SyntaxFactory.GenericName(
                                         DirectCast(newNode, IdentifierNameSyntax).Identifier,
                                         SyntaxFactory.TypeArgumentList(
                                             SyntaxFactory.SeparatedList(typeArguments.Select(Function(p) SyntaxFactory.ParseTypeName(p.ToDisplayParts(typeNameFormatWithGenerics).ToDisplayString()))))) _
                             .WithLeadingTrivia(newNode.GetLeadingTrivia()) _
```

Comment:
please remove the trailing _ s on these lines (if they're no longer necessary).  tnx!